azurerm/data_source_monitor_action_group.go

```diff
@@ -22,13 +24,15 @@ func dataSourceArmMonitorActionGroup() *schema.Resource {
 			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
 
 			"short_name": {
```

Comment:
could we add all of these new blocks [to the documentation](could we add all of these new blocks [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/monitor_action_group.html.markdown#L42)?